services/src/main/java/org/keycloak/services/resources/account/AccountCredentialResource.java

```diff
@@ -92,6 +93,7 @@ public void setLastUpdate(long lastUpdate) {
 
     }
 
+    @JsonIgnoreProperties(ignoreUnknown=true)
```

Comment:
Why is this needed?

Reply:
We need to ignore the password confirmation param as that will be checked on the client.  It's actually quite difficult to remove it from the request without writing lots of terrible, ugly, Angular code.  I didn't see any harm in just letting the REST API ignore it.

Reply:
Password confirmation is the field in the UI that confirms the new password was typed in correctly.  You know, where you ask the user to retype his new password.

I'll change the REST side so that it does this check and throws an error if they don't match.  

The API won't complain if the confirmation is missing.